kolla/node_custom_config/manila.conf

```diff
@@ -0,0 +1,9 @@
+[DEFAULT]
+{% if default_share_type is defined %}
+default_share_type = {{ default_share_type }}
+{% endif %}
+quota_shares = 10
+quota_gigabytes = 2000
+
+[cephfsnfs1]
+cephfs_ganesha_export_ips = {{ ganesha_vlan_interface_address }}
```

Comment:
Does this have a default?

Reply:
fixed with `if ganesha_vlan_interface_address is defined`